src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -1195,10 +1326,21 @@ def setItem(self, item):
         the data range or the histogram of the data using :meth:`setDataRange`
         and :meth:`setHistogram`
         """
-        # While event from items are not supported, we can't ignore dup items
-        # old = self._getItem()
-        # if old is item:
-        #     return
+        old = self._getItem()
+        if old is item:
+            # While event from items are not supported, we can't ignore dup items
```

Comment:
I don't recall what this refers to, I guess it's because there is no even specific event for updates of `getColormappedData` data.

Reply:
Yes